backend/scheduler/tasks.py

```diff
@@ -82,11 +80,15 @@ def execute_pipeline_task(
         return
     logger.info(f"Executing pipeline name: {name}")
     try:
-        logger.info(f"Executing workflow id: {workflow_id}")
         tenant: Organization = (
             get_tenant_model().objects.filter(schema_name=org_schema).first()
         )
         with tenant_context(tenant):
+            pipeline = PipelineProcessor.fetch_pipeline(
+                pipeline_id=pipepline_id, check_active=True
+            )
+            workflow = pipeline.workflow
+            logger.info(f"Executing workflow: {workflow} by pipeline {pipeline}")
```

Comment:
NIT: Does workflow model have a user friendly to_string method? That might help here

Reply:
There have ID and name